app/src/main/java/ca/ubc/best/mint/museandroidapp/vm/FlankerViewModel.java

```diff
@@ -152,36 +196,63 @@ public boolean showArrows() {
     return this.stage == FlankerStage.ARROWS;
   }
 
-  /** @return The color for the left pointer cue. */
+  /** @return The color for the left pointer currCue. */
   public int leftPointerColor() {
     if (!showCue()) {
       return 0;
     }
 
-    boolean isActive = rand.nextBoolean(); // TODO - not random.
-    return isActive ? COLOR_CUE_ON : COLOR_CUE_OFF;
+    if(this.currCue == FlankerCue.NULL) {
```

Comment:
one style suggestion for stuff like this is using switch statements:

switch (this.currCue) {
  case FlankerCue.NULL:
  case FlankerCue.RPP:
    return COLOR_CUE_ON;
  case FlankerCue.LRP:
  case FlankerCue.WARN:
  default:
    return COLOR_CUE_OFF;
}